### DISCUSSION TITLE
YOURLS helm chart installation - shortened urls returns 403 forbidden

### DISCUSSION THREAD

**Discussion by HatimAbdullah** [YOURLS/YOURLS#3722] [Q&A] [answered]
i've deployed YOURLS into my cluster using the [official helm chart](https://github.com/YOURLS/charts) and installed it through admin/install.php endpoint.

after creating my first shortened url and clicked on it i got redirected to the root page (yourl.xxx.xx/) with 403 error.

my values file: 
```
image:
  debug: true

yourls:
  domain: yourls.xxx.xx

service:
  type: ClusterIP

ingress:
  enabled: true
  ingressClassName: "nginx"
  hostname: yourls.xxx.xx
  annotations:
    cert-manager.io/cluster-issuer: letsencrypt-prod
  tls: true

mariadb:
  enabled: false

externalDatabase:
  host: mysql.xxx.svc.cluster.local
  port: 3306
  user: xxx
  password: xxx
  database: yourls
```
error i get on the server when trying to visit the shortened url:
```
[autoindex:error] [pid 34] [client _] AH01276: Cannot serve directory /var/www/html/: No matching DirectoryIndex (index.php,index.html) found, and server-generated directory index forbidden by Options directive
```

i have just started using YOURLS so if there was a specific nginx ingress configuration i need to do please direct me to where i should look.

**Comment by LeoColomb** [comment]
It is normal to get a 403 error when visiting the root path of your installation `yourl.xxx.xx/`.
Did the redirection by itself work? If you create a new short URL to an external URI, does it work?

**Reply by HatimAbdullah** [reply]
No it doesn't work.

The shortened url redirects me to the root page where i get the 403 error.

**Reply by LeoColomb** [reply, marked as answer]
Then it might (should?) be the ingress controller doing the redirection.
Analyzing the HTTP headers of the redirection could help to diagnostic what is answering.

**Reply by HatimAbdullah** [reply]
thanks for your answers, i was able to resolve the issue by going through the HTTP headers.
